src/main/java/uk/gov/hmcts/ccd/domain/service/common/DefaultCaseDataAccessControl.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.ccd.domain.service.common;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.data.SecurityUtils;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.RoleAssignments;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+import uk.gov.hmcts.ccd.domain.service.AccessControl;
+import uk.gov.hmcts.ccd.domain.service.casedataaccesscontrol.RoleAssignmentService;
+
+import java.util.Optional;
+
+@Component
+@ConditionalOnProperty(name = "ccd.new-access-control-enabled", havingValue = "true")
```

Comment:
can you please change this to **enable-attribute-based-access-control** from **ccd.new-access-control-enabled**